lib/services.js

```diff
@@ -450,6 +450,7 @@ export default {
             metadata.putObjectMD(bucketName, longMPUIdentifier,
                 multipartObjectMD, log, (err) => {
                     if (err) {
+                        log.error('error from metadata', { error: err });
```

Comment:
I will remove this "error" log and others that follow the same logic, but not bump any "warn" to "error" in the lower layers because perhaps there are reasons for not logging "error" there. For now, I will note that certain "warn" may need to be updated to "error".